samples/bluetooth/mesh/light_ctrl/src/model_handler.c

```diff
@@ -168,15 +176,22 @@ static const struct bt_mesh_comp comp = {
 
 const struct bt_mesh_comp *model_handler_init(void)
 {
-	int err;
-
 	k_work_init_delayable(&attention_blink_work, attention_blink);
 	k_work_init_delayable(&my_ctx.per_work, periodic_led_work);
 
+	return &comp;
+}
+
+void model_handler_start(void)
+{
+	int err;
+
+	if (bt_mesh_is_provisioned()) {
```

Comment:
So the light ctrl server will store persictently that it should be enabled on boot after it is provisioned?

Reply:
No, that will depend on onpowerup state value: https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/bluetooth/mesh/light_ctrl_srv.c#L1597.

This change will only enable the controller once after the node is provisioned.